server/src/main/kotlin/io/provenance/invoice/domain/dto/InvoiceDto.kt

```diff
@@ -36,4 +38,17 @@ data class InvoiceDto(
             updated = record.updatedTime,
         )
     }
+
+    fun withCalc(calc: InvoiceCalc) = InvoiceDto(
```

Comment:
Nit: This function could be trimmed down to just `this.copy(calc = calc)`

Reply:
ooh, I forgot about that, I'll change it!